app/assets/javascripts/admin/addon/components/ace-editor.js

```diff
@@ -155,7 +155,9 @@ export default Component.extend({
   },
 
   willDestroyElement() {
-    this._darkModeListener.removeListener(this.setAceTheme);
+    if (this._darkModeListener) {
+      this._darkModeListener.removeListener(this.setAceTheme);
+    }
```

Comment:
This caused test failures when a destroy seemingly happened before the listener got set up.